winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -33,7 +33,7 @@ import "windows.storage.idl";
 
 namespace ROOT_NS.AI.MachineLearning 
 {
-    [contractversion(3)]
+    [contractversion(4)]
```

Comment:
please wait for Sheil to sign off on the idl changes.

Reply:
looks good